frontend/javascripts/oxalis/model/sagas/isosurface_saga.ts

```diff
@@ -513,13 +529,23 @@ function* _refreshIsosurfaceWithMap(
   // The isosurface should only be removed once after re-fetching the isosurface first position.
   let shouldBeRemoved = true;
 
+  // todop: store mesh with additional coordinates
```

Comment:
todo: handle this known incompatibility somehow?

Reply:
Good idea! I added the toast into the loadAdHocIsosurface saga.

> Probably, there won't exist any mesh files for >3d datasets. Ad-hoc mesh computation would compute the correct mesh for the current additionalCoordinates but then act like it's the correct one for all other additionalCoordinates, too, right?

yes and yes :+1: